### PR TITLE
Add Go modules support (also: update package import, stabilize tests).

### PR DESCRIPTION
**Pull request by timburks**
This update updates the import path for "OpenAPIv3" to now be lower-case "openapiv3" ([gnostic pr#55](https://github.com/googleapis/gnostic/pull/155)). It also explicitly sorts imported proto dependencies so that they will be rendered in a consistent order (helpful for testing).

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module github.com/googleapis/gnostic-grpc
+
+go 1.12
```
Comment (noahdietz) [inline go.mod]: If possible, can we bump this to go 1.13?

Comment (LorenzHW) [reply go.mod]: Works for me.

Comment (LorenzHW) [reply go.mod]: I have a PR coming up so I am gonna change it to go 1.13 in the upcoming one and merge this one.

Comment (timburks) [reply go.mod]: SGTM

**Pull request merged**